central/networkgraph/entity/datastore/datastore_impl.go

```diff
@@ -492,6 +506,20 @@ func allowed(ctx context.Context, access storage.Access, id string) (bool, error
 	return networkGraphSAC.ScopeChecker(ctx, access).IsAllowed(scopeKey...), nil
 }
 
+func getStoreAccessContext(ctx context.Context, mode storage.Access) context.Context {
```

Comment:
how about having 2 functions one for writes and another for reads to not pass access and make it more readable as line will be shorter
1. `getStoreReadContext`
2. `getStoreWriteContext`

Reply:
#15110